### PR TITLE
bug: #978 add missing edc context values; rename OdrlContext.java to …

### PR DESCRIPTION
**Pull request by ds-lcapellino**
…Context.java

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [x] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

File: irs-edc-client/src/test/java/org/eclipse/tractusx/irs/edc/client/policy/service/EdcPolicyDefinitionServiceTest.java [hunk 1]
```diff
@@ -84,7 +84,10 @@ void testCreatePolicyDefinitionRequest() throws JsonProcessingException, JSONExc
         JSONAssert.assertEquals(objectMapper.writeValueAsString(request), """
                         {
                         	"@context": {
-                        		"odrl": "http://www.w3.org/ns/odrl/2/"
+                        		"odrl": "http://www.w3.org/ns/odrl/2/",
+                        		"@vocab":null,
+                        		"edc":null,
+                        		"cx-policy":null
```
Comment (ds-jhartmann) [inline irs-edc-client/src/test/java/org/eclipse/tractusx/irs/edc/client/policy/service/EdcPolicyDefinitionServiceTest.java:90]: null values in the context are causing errors in EDC. Please exclude the null values.

File: irs-edc-client/src/main/java/org/eclipse/tractusx/irs/edc/client/policy/service/EdcPolicyDefinitionService.java [hunk 2]
```diff
@@ -151,12 +151,11 @@ public EdcCreatePolicyDefinitionRequest createPolicyDefinition(final String poli
...
                                              .build();
 
-        final OdrlContext odrlContext = OdrlContext.builder().odrl(NAMESPACE_ODRL).build();
+        final Context context = Context.builder().odrl(NAMESPACE_ODRL).build();
 
         return EdcCreatePolicyDefinitionRequest.builder()
                                                .policyDefinitionId(accessPolicyId)
-                                               .policy(edcPolicy)
-                                               .odrlContext(odrlContext)
+                                               .policy(edcPolicy).context(context)
```
Comment (ds-jhartmann) [inline irs-edc-client/src/main/java/org/eclipse/tractusx/irs/edc/client/policy/service/EdcPolicyDefinitionService.java:158]: please add the "cx-policy" namespace

**Review by ds-jhartmann — CHANGES_REQUESTED** [review]

File: irs-edc-client/src/main/java/org/eclipse/tractusx/irs/edc/client/policy/service/EdcPolicyDefinitionService.java (hunk 2 above)
Comment (ds-lcapellino) [reply irs-edc-client/src/main/java/org/eclipse/tractusx/irs/edc/client/policy/service/EdcPolicyDefinitionService.java:165]: vocab, edc and cx-policy namespace have been added in the lines above

**Review by ds-jhartmann — APPROVED** [review]
LGTM

**Pull request merged**